content/en/docs/concepts/workloads/controllers/replicaset.md

```diff
@@ -310,6 +310,28 @@ assuming that the number of replicas is not also changed).
 A ReplicaSet can be easily scaled up or down by simply updating the `.spec.replicas` field. The ReplicaSet controller
 ensures that a desired number of Pods with a matching label selector are available and operational.
 
+### Pod Deletion Cost 
+{{< feature-state for_k8s_version="v1.21" state="alpha" >}}
+
+Using the `controller.kubernetes.io/pod-deletion-cost` annotation, users can set a preference regarding 
+which pods to remove first when downscaling a ReplicSet.
```

Comment:
nit: spelling, ReplicaSet

Reply:
I have a couple of questions. You could add a bit more detail.
Is this correct:
The `PodDeletionCost` feature works together with the `controller.kubernetes.io/pod-deletion-cost` annotation
to downscale a ReplicaSet. You can set a numeric preference which determines the order of Pod removal.